OpenAPISpecification.yaml

```diff
@@ -661,6 +661,36 @@ paths:
           description: One of the ids was not found on the server
         '500':
           description: Internal server error
+  /command:
+    post:
+      tags:
+        - Docker Operations
+      summary: Runs a command into a docker container
+      description: >-
+        This command runs a specified command inside a docker container.
+      operationId: command
+      parameters:
+        - in: query
+          name: Id
+          description: The ID of the instance to be deleted
```

Comment:
This seems to be a copy-paste issue. The description should read something like *"The ID of the instance for which the command should be executed"*.